akka-ssl-config/src/main/scala/com/typesafe/sslconfig/akka/AkkaSSLConfig.scala

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package com.typesafe.sslconfig.akka
+
+import java.security.KeyStore
+import java.security.cert.CertPathValidatorException
+import java.util.Collections
+import javax.net.ssl._
+
+import akka.actor._
+import akka.event.Logging
+import com.typesafe.sslconfig.akka.util.AkkaLoggerFactory
+import com.typesafe.sslconfig.ssl._
+import com.typesafe.sslconfig.util.LoggerFactory
+
+object AkkaSSLConfig extends ExtensionId[AkkaSSLConfig] with ExtensionIdProvider {
+
+  //////////////////// EXTENSION SETUP ///////////////////
+
+  override def get(system: ActorSystem): AkkaSSLConfig = super.get(system)
+  def apply()(implicit system: ActorSystem): AkkaSSLConfig = super.apply(system)
+
+  override def lookup() = AkkaSSLConfig
+
+  override def createExtension(system: ExtendedActorSystem): AkkaSSLConfig =
+    new AkkaSSLConfig(system, defaultSSLConfigSettings(system))
+
+  def defaultSSLConfigSettings(system: ActorSystem): SSLConfigSettings = {
+    val akkaOverrides = system.settings.config.getConfig("akka.ssl-config")
```

Comment:
ok, I see that, but that assumption is not enough, since it should be possible to use without akka-stream